### PR TITLE
AIHW My Hospitals (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 

image
image
image

- [x ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2817 @@
...
+        "description": "Gets a measure download, in XLSX format, for the supplied measure download code.",
+        "operationId": "GetMeasureDownload"
+      }
+    },
+    "/measure-downloads/across-reporting-units/{measure-download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2817 @@
...
+          }
+        ]
+      }
+    },
+    "/reporting-units-downloads/mappings": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2817 @@
...
+          }
+        ]
+      }
+    },
+    "/reporting-units-downloads/{datasheet-code}/{reporting-unit-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2817 @@
...
+          }
+        ]
+      }
+    },
+    "/simple-downloads/download-codes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2817 @@
...
+          }
+        ]
+      }
+    },
+    "/simple-downloads1/{download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2817 @@
...
+        "description": "Gets formatted data in a flattened structure with each item corresponding to a single data point. Data is restricted to within a single measure category and can be optionally filtered by measure, reporting unit code, reporting unit type and reporting dates. Matching within each filter is by logical disjunction (OR) and matching across filters is by logical conjunction (AND). NOTE: There is currently a pagination restriction on this query that restricts requests to a maximum of 1000 results. Use skip and top to control this pagination. Sorting is not currently available.",
+        "operationId": "GetFlatFormattedData"
+      }
+    },
+    "/flat-data-extract/{measure-category-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,65 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiProperties.json:4]: @paulculmsee Please go thru the [template provided for apiProperties.json file for IP connectors](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json). The iconBrandColor must be "#da3b01".

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,2817 @@
...
+          }
+        ]
+      }
+    },
+    "/measure-downloads1/{measure-download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 3 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 4 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 5 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 6 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [inline independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 8 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @sriyen-msft  content type is application/vnd.openxmlformats-officedocument.spreadsheetml.sheet so can there be a schema here?

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 2 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @sriyen-msft content type is application/vnd.openxmlformats-officedocument.spreadsheetml.sheet so can there be a schema here?

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 3 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: Same issue as above here @sriyen-msft

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 4 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: Same issue as above in this endpoint too @sriyen-msft

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 5 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: Same issue as above in this endpoint too @sriyen-msft

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 6 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: This one is my oversight... will be fixed in next commit

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 1 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: @sriyen-msft content type is application/vnd.openxmlformats-officedocument.spreadsheetml.sheet so can there be a schema here?

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 6 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json (hunk 3 above)
Comment (paulculmsee) [reply independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json:1630]: @sriyen-msft content type is application/vnd.openxmlformats-officedocument.spreadsheetml.sheet so can there be a schema here?

**Comment by sriyen-msft** [comment]
@paulculmsee Sorry for the delay in response, We do not have a schema definition idea for the content type you have shared, Let me move forward and prepare your connector for production deployment. We may receive suggestions from the engineering team during prepping your connector for prod deployment. Thanks

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by sriyen-msft** [comment]
@paulculmsee Sorry for the delay in response, your connector is now in production deployment and reached to [Asia](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions by Friday 11th February 2022. Please let us know if you have any questions, Thanks.

**Comment by sriyen-msft** [comment]
@paulculmsee Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**